testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/reactive/messaging/kafka/ssl/microprofile-config-ssl-global.properties

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright The WildFly Authors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# 9092 is secured with SSL as per ReactiveMessagingKafkaSslTestCase
+bootstrap.servers=[localhost:9092]
+mp.messaging.connector.smallrye-kafka.wildfly.elytron.ssl.context=kafka-ssl-test
```

Comment:
Note to self or future maintainers. Testing this being configured here rather than in mp.messaging.outgoing.to-kafka.wildfly.elytron.ssl.context=kafka-ssl-test and mp.messaging.incoming.from-kafka.wildfly.elytron.ssl.context=kafka-ssl-test is what this PR is all about.